packages/web/src/common/store/pages/profile/lineups/tracks/sagas.js

```diff
@@ -43,8 +51,13 @@ function* getTracks({ offset, limit, payload, handle }) {
   )
   const sort = payload?.sort === TracksSortMode.POPULAR ? 'plays' : 'date'
   const getUnlisted = true
+  const getFeatureEnabled = yield getContext('getFeatureEnabled')
+  const readArtistPickFromDiscoveryEnabled = yield call(
+    getFeatureEnabled,
+    FeatureFlags.READ_ARTIST_PICK_FROM_DISCOVERY
+  ) ?? false
 
-  if (user._artist_pick) {
+  if (!readArtistPickFromDiscoveryEnabled && user._artist_pick) {
```

Comment:
Discovery API will return the artist pick track 1st based on the `artist_pick_track_id` in the `users` table so this reordering logic isn't necessary when reading/writing the artist pick to discovery